browser/src/index.tsx

```diff
@@ -165,11 +178,25 @@ const start = (args: string[]) => {
         UI.setMode(newMode)
 
         if (newMode === "normal") {
+            if (cursorLine) { // TODO: Add "unhide" i.e. only show if previously visible
```

Comment:
The TODO makes sense. I'm thinking we could have an event like Config.on("changed", (newConfigKey, newConfigValue) => { ... }) where we could add this. This could happen if we reload the config file, or if we synchronize config values from Neovim, etc. No changes needed, just thinking out loud